rero_ils/modules/stats_cfg/jsonschemas/stats_cfg/stat_cfg-v0.0.1.json

```diff
@@ -0,0 +1,149 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "type": "object",
+  "title": "Statistics report configuration",
+  "description": "JSON schema for statistics report configuration.",
+  "additionalProperties": false,
+  "required": [
+    "$schema",
+    "pid",
+    "indicator",
+    "dist1",
+    "dist2",
```

Comment:
Distributions should not be `required`. I can have zero, one, or two distributions in a report.